src/librustdoc/visit_ast.rs

```diff
@@ -330,7 +335,7 @@ impl<'a, 'tcx> RustdocVisitor<'a, 'tcx> {
                 }
             }
             // If we're inlining, skip private items or item reexported as "_".
```

Comment:
This comment probably needs to be updated

Reply:
Good catch!